### PR TITLE
[master] Lesson14

### PR DESCRIPTION
**Pull request by spanteleyko**

### CODE REVIEW

File: Lesson14/src/com/gmail/panser81/lesson14/models/Product.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.gmail.panser81.lesson14.models;
+
+public class Product {
+    private String name;
+    private String uniqueNumber;
+    private double price;
```
Comment (aperavoznikau-idt) [inline Lesson14/src/com/gmail/panser81/lesson14/models/Product.java:6]: Please use BigDecimal

File: Lesson14/src/com/gmail/panser81/lesson14/services/impl/CreateFileLessonServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        StringBuilder productsBuilder = new StringBuilder();
+
+        productsBuilder.append("Name;UniqueNumber;Price;Count;Production\n");
+        productsBuilder.append("Windows 10;00001;2000;20;Microsoft\n");
+        productsBuilder.append("Usb flash drive;4561;500;100;Samsung\n");
+        productsBuilder.append("Dell x-01;200232;1000;3500;DELL\n");
+        productsBuilder.append("Dell ODF-01;3459834;5000;10;DELL\n");
+        productsBuilder.append("ASUS 5000;45600001;980;150;ASUS\n");
+
+        String filePath = LessonConstants.DIRECTORY_NAME + "/" + LessonConstants.FILE_NAME;
```
Comment (aperavoznikau-idt) [inline Lesson14/src/com/gmail/panser81/lesson14/services/impl/CreateFileLessonServiceImpl.java:22]: Please use String.format

**Review by aperavoznikau-idt — CHANGES_REQUESTED** [review]

File: Lesson14/src/com/gmail/panser81/lesson14/models/Product.java (hunk 1 above)
Comment (spanteleyko) [reply Lesson14/src/com/gmail/panser81/lesson14/models/Product.java]: changed

File: Lesson14/src/com/gmail/panser81/lesson14/services/Mapper.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.gmail.panser81.lesson14.services;
+
+import com.gmail.panser81.lesson14.models.Product;
+
+import java.util.List;
+
+public interface Mapper {
+    List<Product> Map(String productsStr);
```
[the remark this answers is not in the document]
Comment (spanteleyko) [reply Lesson14/src/com/gmail/panser81/lesson14/services/Mapper.java]: fixed

File: Lesson14/src/com/gmail/panser81/lesson14/services/impl/CreateFileLessonServiceImpl.java (hunk 2 above)
Comment (spanteleyko) [reply Lesson14/src/com/gmail/panser81/lesson14/services/impl/CreateFileLessonServiceImpl.java]: done

**Pull request merged**